client/ayon_blender/api/ops.py

```diff
@@ -186,9 +187,9 @@ def _process_app_events() -> Optional[float]:
 
         # Refresh Manager
         if GlobalClass.app:
-            manager = GlobalClass.app.get_window("WM_OT_avalon_manager")
-            if manager:
-                manager.refresh()
+           manager = GlobalClass.app.get_window("WM_OT_avalon_manager")
+           if manager:
+               manager.refresh()
```

Comment:
Is this indentation wrong?
```suggestion
            manager = GlobalClass.app.get_window("WM_OT_avalon_manager")
            if manager:
                manager.refresh()
```